serverless-plugin/index.js

```diff
@@ -258,15 +242,14 @@ class ServerlessPlugin {
             ...ADDITIONAL_EXCLUDE,
         ];
 
-        // Each package will have its own wrapper; remember its options to add
-        // to the handler template
-        const handlerOptions = {};
-
         // Keep track of which extra libraries were copied
         const libraries = {};
 
         let haskellFunctionsFound = false;
 
+        // We run stack clean, so the TemplateHaskell gets regenerated in case it is outdated
+        this.runStack("", ['clean']);
```

Comment:
We could remove this, but take into account that a user could add a new handler, and if `Main.hs` is not rebuilt, it won't be included into the dynamic dispatcher. It could generate a lot of confusion, so I decided to include the `clean` command. You won't be deploying in short periods of time, and given that the deployment itself takes some time, the additional time of building the project is not that noticeable.

Reply:
Since the modules for all the handlers need to be imported into `Main`, won't it be recompiled in that case?

Reply:
This is true, but there are cases where it isn't recompiled, that's why we opted for running a clean each time.